### ISSUE TITLE
rewording required

### ISSUE THREAD

**Issue by krisjpi** [commons-stack/tech-dapp#149]
We cannot have the phrase "Access to Future ABC Hatch Phases" under the "for your contribution" section. We can reword it to be more accurate.

**Comment by tamarandom** [comment]
@krisjpi Good point. Will you please include the exact wording that should be used. I can think of no one better suited to suggest the legally appropriate language than you. ;)

**Comment by tamarandom** [comment]
Closing! Dupe of #132!
